script.module.dropbox/resources/language/German/strings.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```

Comment:
xml strings are outdated, please move to the new language system.
http://kodi.wiki/view/language_support